grafana-plugin/src/components/Text/Text.tsx

```diff
@@ -81,8 +83,10 @@ const Text: TextInterface = (props) => {
     setValue(e.target.value);
   }, []);
 
+  const CustomTag = (customTag || `span`) as unknown as React.ComponentType<any>;
```

Comment:
I think now we can get rid of casting here:
```
const CustomTag = customTag
```

And in line 60 apply default prop
```
...,
clickable,
customTag = 'span',
```